lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -440,10 +441,19 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   end
 
   @impl true
-  def handle_pad_added(Pad.ref(:output, {track_id, _rid}) = pad, _ctx, state) do
-    %{encoding: encoding} = Map.get(state.inbound_tracks, track_id)
+  def handle_pad_added(Pad.ref(:output, {track_id, rid}) = pad, ctx, state) do
+    %Track{encoding: encoding} = track = Map.get(state.inbound_tracks, track_id)
     extensions = Map.get(state.extensions, encoding, []) ++ Map.get(state.extensions, :any, [])
 
+    link_to_track_sender =
+      if Map.has_key?(ctx.children, {:track_sender, track_id}) do
+        fn link_builder -> to(link_builder, {:track_sender, track_id}) end
+      else
+        fn link_builder ->
+          to(link_builder, {:track_sender, track_id}, %TrackSender{track: track})
+        end
```

Comment:
I think using capture here won't hurt readability and will make it more concise
```suggestion
        &to(&1, {:track_sender, track_id})
      else
        &to(&1, {:track_sender, track_id}, %TrackSender{track: track})
```